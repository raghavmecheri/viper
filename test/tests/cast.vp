int b = toInt(9.34);
string c = toString(888);
string d = toString(false);
bool e = toBool(400);
bool f = toBool("");
float h = toFloat(8);
char j = toChar(255);

int a2 = 4;
string k = toString(a2);
