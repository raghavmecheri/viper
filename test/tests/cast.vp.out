9
99
888
false
1
0
8
d
h